licenses.html

```diff
@@ -9215,6 +9409,158 @@ <h4>Used by:</h4>
                 <pre class="license-text">This project is dual-licensed under the Unlicense and MIT licenses.
 
 You may use this code under the terms of either license.
+</pre>
+            </li>
+            <li class="license">
+                <h3 id="MPL-2.0">Mozilla Public License 2.0</h3>
+                <h4>Used by:</h4>
+                <ul class="license-used-by">
+                    <li><a href=" https://github.com/vorner/signal-hook ">signal-hook-registry</a></li>
```

Comment:
I'm confused / need help.  This line seems to indicate that the MPL-2.0 license is used by [`signal-hook-registry`](https://crates.io/crates/signal-hook-registry), which states both on its Crates profile (linked previously) and on its repo [here](https://github.com/vorner/signal-hook/blob/master/signal-hook-registry/Cargo.toml) and [here](https://github.com/vorner/signal-hook/tree/master/signal-hook-registry) that it's MIT OR Apache 2.0, no? (At least, according to the human-readable text.)

That said, is the actual specifier in the `Cargo.toml` correct?  It seems that the correct notation should be [`MIT OR Apache-2.0`](https://github.com/rust-lang/cargo/blob/489b66f2e458404a10d7824194d3ded94bc1f4e4/Cargo.toml#L5), but that `signal-hook-registry` is using [`Apache-2.0/MIT`](https://github.com/vorner/signal-hook/blob/d4bd2bab6fc8b65d5e0b917f8c248addaab51ebf/signal-hook-registry/Cargo.toml#L14).

The usage of a `/` is not my understanding of a valid SPDX identifier.  Instead, I believe that `OR` and `AND` and `WITH` would be acceptable, according to [Appendix IV of the SPDX reference](https://spdx.dev/spdx-specification-21-web-version/#h.jxpfx0ykyb60)

Maybe that's askew triggering a false alert here?  Or alternative theory, are we actually using the latest version?

Reply:
Turns out it was another bug in license tooling which is now fixed upstream.  The MPL-2 license addition is real and on account of [`smartstring`](https://github.com/bodil/smartstring).  I'll resolve this comment.